cg/meta/workflow/balsamic.py

```diff
@@ -86,7 +95,10 @@ def get_deliverables_file_path(self, case_id: str, check_exists: bool = False) -
 
     def get_config_path(self, case_id: str, check_exists: bool = False) -> str:
         """Generates a path where the Balsamic config for the case_id should be located.
-        (Optional) Checks if config file exists."""
+
+        (Optional) Checks if config file exists.
+        """
+        LOG.info("Fetch balsamic config file for %s", case_id)
```

Comment:
Same with this, its getting the path not only to use the config file, but to tell the balsamic where to save it aswell! So might be misleading. You also will be calling this to get timestamp and pipeline from config so this will be logged on the screen a whole bunch!

Reply:
Changed